azure-pipelines.yml

```diff
@@ -320,7 +320,12 @@ stages:
           image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-14.04-cross-0cd4667-20170319080304
           options: --init # This ensures all the stray defunct processes are reaped.
         pool:
+          ${{ if eq(variables['System.TeamProject'], 'public') }}:
           vmImage: ubuntu-18.04
+          ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+            name: NetCore1ESPool-Internal
+            demands: ImageOverride -equals Build.Ubuntu.1804.Amd64
```

Comment:
It would be nice if we could start enumerating our demands for image content, even if we still have to specify a particular image for now. That way we're documenting the parts of an image we use, and if the image were to change to omit things we needed we would know early.